internal/output/html/style.css

```diff
@@ -269,11 +269,31 @@ header {
   color: #fff;
 }
 
+.osv-tab {
+  position: relative;
+}
+
 .osv-tab iframe {
   width: 100%;
   min-height: 1000px;
   border: none;
-  background-color: #000;
+}
+
+.iframe-spinner  {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
```

Comment:
I believe you can use [`inset`](https://developer.mozilla.org/en-US/docs/Web/CSS/inset) here:

```suggestion
  inset: 0;
```

Reply:
SGTM!